Source/BugsnagClient.m

```diff
@@ -423,10 +424,12 @@ - (void)start {
 
     [self.sessionTracker startNewSessionIfAutoCaptureEnabled];
 
-    [self addBreadcrumbWithBlock:^(BugsnagBreadcrumb *_Nonnull breadcrumb) {
-        breadcrumb.type = BSGBreadcrumbTypeState;
-        breadcrumb.message = BSGBreadcrumbLoadedMessage;
-    }];
+    if ([self.configuration shouldRecordBreadcrumbType:BSGBreadcrumbTypeState]) {
```

Comment:
It may be worth creating a method called something like `addAutoBreadcrumb`  which encapsulates this behaviour and reduces the risk of future automatic breadcrumbs forgetting to add this check. [This](https://github.com/bugsnag/bugsnag-android/pull/842/files#diff-d01d476975cb3e9ebc74fb3cec71c943R824) is what the Android implementation has done.

Reply:
After discussion It was agreed to make the suggested change as part of #486.